yb-voyager/cmd/export.go

```diff
@@ -388,32 +387,28 @@ func saveExportTypeInMSR() {
 	}
 }
 
-func checkDependenciesForExport() error {
-	var errs []string
+func checkDependenciesForExport() (problems []string, err error) {
```

Comment:
rename the `problems` to `binaryCheckIssues`  and `problem` to `binaryCheckIssue` or similar to `issue`